tests/acceptance/stepDefinitions/alertsContext.js

```diff
@@ -0,0 +1,6 @@
+const { client } = require('nightwatch-api')
+const { Then } = require('cucumber')
+
+Then('the user closes the alert', function () {
```

Comment:
```suggestion
When('the user closes the alert', function () {
```
It's a action that you want to test, not a result, so it should be a `When` step